src/mflux_debugger/debugger_service.py

```
@@ -0,0 +1,1933 @@
+                    success=True,
+                    message=message,
+                    data=response_data,
+                )
+            else:
+                return DebuggerResponse(success=True, message="⏸️ Program is paused")
+        elif self.debugger.state == DebugState.FINISHED:
+            return DebuggerResponse(success=True, message="✅ Program finished")
+        else:
+            return DebuggerResponse(success=True, message="▶️ Program is running")
+
+    def _sync_mlx_if_available(self, timeout: float = 10.0) -> bool:
```

Ruff (0.14.3)

Unused method argument: `timeout`

(ARG002)